roles/ipaclient/vars/default.yml

```diff
@@ -2,4 +2,4 @@
 # vars/default.yml
 ---
 ipaclient_packages: [ "ipa-client", "python3-libselinux" ]
-#ansible_python_interpreter: '/usr/bin/python3'
+# ansible_python_interpreter: '/usr/bin/python3'
```

Comment:
I think the ansible_python_interpreter lines can be removed. This is handled by Ansible.

Reply:
Fixed.